Automattic/wp-calypso
# Component: Add Jetpack Scan Product Icon

#### Changes proposed in this Pull Request

* Adds the Jetpack Scan product Icon. 
<img width="940" alt="Screen Shot 2020-03-26 at 1 04 20 PM" src="https://user-images.githubusercontent.com/115071/77645031-57477380-6f62-11ea-8617-07ad901a4b23.png">


#### Testing instructions
* Run `npm run components:storybook:start`
* Navigate to http://localhost:62269/?path=/story/producticon--default 

Notice that the new icon shows up and that non of the other icons break. 

Fixes #
